.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: python
+python:
+  - "3.8"
```

Comment:
@tzumainn what's the version of python we should be using? I set it to 3.8 since that's what I had installed on my local machine.

Reply:
I'm actually unsure myself, but your PR doesn't break my checks on 3.7, so I'm fine with it! It can always be adjusted later.